compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -2023,6 +2028,25 @@ private void validateStreamingActionFunctionParameters(BLangStreamAction streamA
         }
     }
 
+    private void resolveStreamingScopeEntries(SymbolEnv stmtEnv, StreamingQueryStatementNode node) {
```

Comment:
shall we rename to `defineQueryParams` or similar? feels thats what it actually does..